Bottom: 4bbcb14e0681aad0a350fedd37f6fa80dd36ffe4
Top:    58875e42505b600ff8e806fb04b1bfe5d57dee6b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 23:34:51 +0100

Refresh of move-c-fix-incorrect-usage-of

---

diff --git a/src/move.c b/src/move.c
index b50531e..c9e8628 100644
--- a/src/move.c
+++ b/src/move.c
@@ -910,7 +910,7 @@ move_all_thermals (int old_index, int new_index)
 static int
 LastLayerInTopGroup (int layer)
 {
-  int top_group = GetLayerGroupNumberByNumber(max_group + TOP_LAYER);
+  int top_group = GetLayerGroupNumberByNumber(top_silk_layer);
   int lgroup = GetLayerGroupNumberByNumber(layer);
   if (top_group == lgroup
       && PCB->LayerGroups.Number[lgroup] == 2)
@@ -921,7 +921,7 @@ LastLayerInTopGroup (int layer)
 static int
 LastLayerInBottomGroup (int layer)
 {
-  int bottom_group = GetLayerGroupNumberByNumber(max_group + BOTTOM_LAYER);
+  int bottom_group = GetLayerGroupNumberByNumber(bottom_silk_layer);
   int lgroup = GetLayerGroupNumberByNumber(layer);
   if (bottom_group == lgroup
       && PCB->LayerGroups.Number[lgroup] == 2)
